pages/emailSender.tsx

```diff
@@ -124,30 +124,30 @@ const EmailSender: NextPage = () => {
     return headersArrFinal
   }
 
-  const sendEmails = () => {
-    const localErrorMessages: ErrorMessage[] = []
-    for (let i = 0; i < finalMessages.length; i++) {
-      // This if statement is just for testing / a mock. You can read this as if (error)
-      // to represent the error case. Replace the following line when actually implementing
-      // the error check.
-      if (i % 2 === 0) {
-        localErrorMessages.push({
-          id: finalMessages[i].id,
-          message: 'Errorrrr!'
-        })
-      }
-    }
-    setResultErrorMessage({
-      errorMessages: localErrorMessages,
-      resultMessage: {
-        isError: localErrorMessages.length > 0,
-        message:
-          localErrorMessages.length > 0
-            ? `Error sending ${localErrorMessages.length} of ${finalMessages.length}`
-            : 'Sent emails successfully'
-      }
-    })
-  }
+  // const sendEmails = () => {
```

Comment:
unsure if we even need local error messages - i'll work on making our error checking system better next week if that works

Reply:
Sounds good!